features/data/successful_tests.xml

```diff
@@ -15,4 +15,11 @@
      <param pos="1" name="os.version" />
      <param pos="0" name="os.name" value="Bar" />
    </fingerprint>
+   <fingerprint pattern="^foo sb\-([\d.]+)$">
```

Comment:
From my testing [features/verify.feature](https://github.com/rapid7/recog/blob/main/features/verify.feature#L15) fails and needs to be updated since this adds a 5th example and changes the output message.

```
  Scenario: Successful tests                       # features/verify.feature:11
    When I run `recog_verify successful_tests.xml` # aruba-2.1.0/lib/aruba/cucumber/command.rb:3
    Then it should pass with exactly:              # aruba-2.1.0/lib/aruba/cucumber/command.rb:404
      """
      successful_tests.xml: SUMMARY: Test completed with 4 successful, 0 warnings, and 0 failures
      """
      expected `recog_verify successful_tests.xml` to have output output string is eq: "successful_tests.xml: SUMMARY: Test completed with 4 successful, 0 warnings, and 0 failures"
      but was:
         successful_tests.xml: SUMMARY: Test completed with 5 successful, 0 warnings, and 0 failures
      Diff:
      @@ -1 +1 @@
      -(output string is eq: "successful_tests.xml: SUMMARY: Test completed with 4 successful, 0 warnings, and 0 failures")
      +"successful_tests.xml: SUMMARY: Test completed with 5 successful, 0 warnings, and 0 failures"
       (RSpec::Expectations::ExpectationNotMetError)
      features/verify.feature:13:in `it should pass with exactly:'
```

Reply:
Addressed, thanks. After this morning's push the only errors there are due to the changes made in `recog-ruby`

```
  @no-clobber
  Scenario: Successful tests                       # features/verify.feature:11
    When I run `recog_verify successful_tests.xml` # aruba-2.1.0/lib/aruba/cucumber/command.rb:3
    Then it should pass with exactly:              # aruba-2.1.0/lib/aruba/cucumber/command.rb:404
      """
      successful_tests.xml: SUMMARY: Test completed with 5 successful, 0 warnings, and 0 failures
      """
      Expected `recog_verify successful_tests.xml` to succeed but got non-zero exit status and the following output:
      
      successful_tests.xml:18: FAIL: 'test of temp params' is missing an example that checks for parameter '_tmp.001' which is derived from a capture group
      successful_tests.xml: SUMMARY: Test completed with 5 successful, 0 warnings, and 1 failures
      
       (RSpec::Expectations::ExpectationNotMetError)
      features/verify.feature:13:in `it should pass with exactly:'
 ```